awscli/customizations/wizard/ui/layout.py

```diff
@@ -278,3 +292,43 @@ def _create_buttons_key_bindings(self, buttons):
 
     def __pt_container__(self):
         return self.container
+
+
+class WizardErrorBar:
```

Comment:
So I think exiting the dialog depends on the error that you get back. So if it was something messed up with the input, it makes sense to go back. But if it is something that can be changed asynchronously from the wizard say there is already a role created with that name you have to delete or you are missing the correct permissions to execute the wizard and need to add it, it may not necessarily make sense to go back.

In terms of controlling the error buffer, I was even thinking we just clear the error buffer once we update an answer to any of the prompts. I can also see maybe making toggling of the error bar as an option. I just want to avoid cluttering/adding complexity to the toolbar if we do not need to. I don't really have strong opinions on it other than I do agree clearing the error immediately after exiting may not be the best option.